.github/workflows/unit_tests.yml

```diff
@@ -29,11 +29,19 @@ on:
         default: ""
       linux_nightly_6_0_enabled:
         type: boolean
-        description: "Boolean to enable the Linux nightly 6.0 Swift version matrix job. Defaults to true."
+        description: "⚠️  Deprecated, we no longer run against 6.0 nightly."
```

Comment:
Should these defaults be changed?

Reply:
I don’t think it matters, we just drop it on the floor now